internal/pkg/workspace/workspace.go

```diff
@@ -585,25 +605,55 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 	return dockerfiles, nil
 }
 
+func (ws *Workspace) manifestNameMatchWithDir(mft namedManifest, mftDirName string) error {
```

Comment:
i like this, we should do it for workloads if we don't already

Reply:
we are!